azurelinuxagent/common/utils/archive.py

```diff
@@ -0,0 +1,213 @@
+# Microsoft Azure Linux Agent
+#
+# Copyright 2018 Microsoft Corporation
```

Comment:
Drop the year from the copyright statement

Reply:
I switched to the approve style for these files.